MyOwnORM/MyOwnORM/Services/CustomDbSetReflection.cs

```diff
@@ -0,0 +1,514 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using MyOwnORM.Attributes;
+using MyOwnORM.Helper;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM.Reflection
+{
+    public class CustomDbSetReflection<T> : CustomDbSetReflectionHelper<T> where T : class
```

Comment:
Now I can see that CustomDbSetReflection and CustomDbSetReflectionHelper work for 1 purpose, so you couldn't inherit one of another. I can propose to union them(and make methods needed only to CustomDbSetReflection private) or inject CustomDbSetReflectionHelper to CustomDbSetReflection to use it as usual helper,